README.md

```diff
@@ -63,3 +63,14 @@ Please don't write too much code in the starting position or add too many unit t
 
 If your programming language doesn't have an easy way to add a command-line interface, then the TextTest fixture is probably not necessary.
 
+### Recommended project structure
```

Comment:
I tried to mimic the language above, e.g. 'update_quality' versus `update_quality`